app/Providers/AppServiceProvider.php

```diff
@@ -16,7 +17,9 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        $this->app->bind(RequestActivityLoggerInterface::class, DummyRequestActivityLogger::class);
+        $this->app->bind(RequestActivityLoggerInterface::class, function (){
+            return $this->app->make(env('APP_ENV') != 'production' ? DebugRequestActivityLogger::class : ProductionRequestActivityLogger::class);
```

Comment:
env helper won't work in the cached state of application so you need to use instead configs.

Reply:
changed